views/includes/page-templates/beneficial-owner.html

```diff
@@ -37,7 +37,7 @@ <h1 class="govuk-heading-xl govuk-!-margin-0">{{ title }}</h1>
 
       <button class="govuk-button" data-module="govuk-button" id="add" formaction="{{ addButtonAction }}" >Add<span class="govuk-visually-hidden"> {{ whatToAdd }} </span></button>
 
-      {% if (hasBeneficialOwners or hasManagingOfficers) %}
+      {% if ((hasBeneficialOwners or hasManagingOfficers) and entity_name) %}
```

Comment:
This should be using entity_number which is in update journey. If only for registration then use "and not entity_number"

Reply:
My bad, meant to prevent rendering list on validation error. Please review recent commit to fix